spec/lexers/kotlin_spec.rb

```diff
@@ -22,5 +22,21 @@
     it 'recognizes one-line comments not followed by a newline (#797)' do
       assert_tokens_equal '// comment', ['Comment.Single', '// comment']
     end
+
+    it 'recognizes destructured assignment' do
+      assert_tokens_equal 'val (a, b) = pair',
+        ['Keyword.Declaration', 'val'],
+        ['Text', " "],
+        ['Punctuation', "("],
+        ['Name.Property', 'a'],
+        ['Punctuation', ","],
+        ['Text', " "],
+        ['Name.Property', 'b'],
+        ['Punctuation', ")"],
+        ['Text', " "],
+        ['Punctuation', "="],
+        ['Text', " "],
+        ['Name', "pair"]
+    end
```

Comment:
Thanks for the tests. However, this isn't necessary. This makes tests brittle and we have dynamic tests that check for errors based on the `samples` provided. You've provided a sample for this case so that's perfect. Please remove this test.